doc/source/building/index.rst

```diff
@@ -353,7 +353,7 @@ virtual environments:
     PyPI with::
 
        # Build dependencies
-       python -m pip install numpy cython pythran pybind11 meson ninja pydevtool rich-click
+       python -m pip install numpy cython pythran pybind11 meson-python ninja pydevtool rich-click
```

Comment:
Does it install `meson` itself through `meson-python` but not `ninja`?

Reply:
Yes indeed.